docs/troubleshooting.rst

```diff
@@ -139,15 +139,78 @@ Errors from the terminal when Mantid Imaging is launched, such as:
 
 These are harmless and can be ignored.
 
+UserWarning: CUDA path could not be detected. Set CUDA_PATH environment variable if CuPy failed to load
+----------------------------------------------------------------------------------------------------------
+This warning likely indicates that the  :code:`CUDA_PATH` environment variable is not set correctly, or that the CUDA Toolkit is not installed on your system.
 
+If Mantid Imaging is running correctly, you can ignore this warning. However, if you are experiencing issues with CUDA-based algorithms such as PDHG, you may need to set the  :code:`CUDA_PATH` environment variable to the path where CUDA Toolkit is installed on your machine. Instructions for setting environment variables can be found below under the  :ref:`ImportError: DLL load failed while importing astra_c <importerror-dll-load-failed-while-importing-astra_c-the-specified-module-could-not-be-found>` section.
+
+.. _importerror-dll-load-failed-while-importing-astra_c-the-specified-module-could-not-be-found:
 
 ImportError: DLL load failed while importing astra_c: The specified module could not be found
 ---------------------------------------------------------------------------------------------
 
-This error can arise when the CUDA-version of :code:`astra-toolbox` is installed on a machine that doesn't have a GPU.
-By default, the CUDA-version of :code:`astra-toolbox` is installed during environment setup.
+This error usually means that :code:`astra-toolbox` is not installed correctly or the CUDA Runtime libraries are missing.
 
-Suggested Fix
-^^^^^^^^^^^^^
+By default, the CUDA-version of :code:`astra-toolbox` is installed during environment setup, **HOWEVER** it is possible for the incorrect version to be installed for non-NVIDIA GPUs or for machines without a GPU. You can check which version is installed by running the command :code:`mamba list astra-toolbox` in a terminal and verifying that the version installed is the python or cuda version of the package. If the CUDA-version is installed, but you do not have a compatible GPU or the CUDA Runtime libraries installed, you will likely encounter this error.
+
+If you are still experiencing this error after verifying that you have a compatible GPU and the CUDA Runtime libraries installed, it is possible that `CUDA Toolkit 12.9 <https://developer.nvidia.com/cuda-downloads?target_os=Windows&target_arch=x86_64&target_version=11&target_type=exe_network>`_ is not installed correctly or the environment variable  :code:`CUDA_PATH` is not set correctly.
+
+You can check if Windows can find your CUDA device drivers are present by opening a terminal and entering the command:  :code:`nvidia-smi` you should see an  output similar to the following:
+
+.. table:: Example output from nvidia-smi
+
```

Comment:
The table formatting seems a bit off to me, maybe better as a code block
```
.. code-block:: none

   +-----------------------------------------------------------------------------------------+
   | NVIDIA-SMI 576.57                 Driver Version: 576.57         CUDA Version: 12.9     |
   +-----------------------------------------+------------------------+----------------------+
```

Reply:
I get a horizontal scroll box rather than wrapping on Firefox.

Or we could drop showing the output. It not hard for the user to see the Cuda version in the output.

Reply:
I'll see if i can resolve the last few formatting issues with the table and if not, I can drop. interesting you should get a horizontal scroll on Firefox as i also used Firefox.